magit-file-icons.el

```diff
@@ -88,11 +128,17 @@
    (magit-file-icons-mode
     (when magit-file-icons-enable-diff-file-section-icons (el-patch-eval-template #'magit-diff-insert-file-section 'defun))
     (when magit-file-icons-enable-untracked-icons (el-patch-eval-template #'magit-insert-untracked-files 'defun))
-    (when magit-file-icons-enable-diffstat-icons (el-patch-eval-template #'magit-diff-wash-diffstat 'defun)))
+    (when magit-file-icons-enable-diffstat-icons (el-patch-eval-template #'magit-diff-wash-diffstat 'defun))
+    (when magit-file-icons-enable-diff-file-section-icons (magit-file-icons-refresh-backend))
+    (when magit-file-icons-enable-untracked-icons (magit-file-icons-refresh-backend))
+    (when magit-file-icons-enable-diffstat-icons (magit-file-icons-refresh-backend)))
```

Comment:
I think it's better to combine these into an `or` clause, or just run the refresh unconditionally
```suggestion
    (magit-file-icons-refresh-backend)
```